### DISCUSSION TITLE
set_timer usage

### DISCUSSION THREAD

**Discussion by drticcz** [tasmota/Berry_playground#32] [Get Help - Q&A] [answered]
I cannot wrap my head around why this extremely simple code prints out "Start" and "stop" within 10ms instead of the 5s as I want it to... any explanation?

I run it from the berry console, Tasmota 13.2

```
def dummy()
    return
end

print("start")
tasmota.set_timer(5000, dummy)
print("stop")

```

**Comment by darrylb123** [comment, marked as answer]
Move the print("Stop") to inside the dummy() function. That is the code
that is run when the timer expires.
